packages/discord.js/src/structures/AttachmentBuilder.js

```diff
@@ -16,16 +16,45 @@ class AttachmentBuilder {
      * @type {BufferResolvable|Stream}
      */
     this.attachment = attachment;
+
     /**
      * The name of this attachment
      * @type {?string}
      */
     this.name = data.name;
+
     /**
      * The description of the attachment
      * @type {?string}
      */
     this.description = data.description;
+
+    /**
+     * The title of the attachment
+     * @type {?string}
+     */
+    this.title = data.title;
+
+    /**
+     * The base64 encoded byte array representing a sampled waveform
+     * @type {?string}
+     */
+    this.waveform = data.waveform;
+
+    /**
+     * The duration of the attachment in seconds
+     * @type {?number}
+     */
+    this.duration = data.duration;
+  }
+
+  /**
+   * Whether this attachment has been marked as a spoiler
+   * @type {boolean}
+   * @readonly
+   */
+  get spoiler() {
```

Comment:
This makes no sense in a builder, I wonder why we even kept it LOL

Reply:
Yeah... I just moved it up since it was lost in between the other methods